src/renderers/BaseRenderer.js

```diff
@@ -1252,6 +1264,48 @@ export default class BaseRenderer extends EventEmitter {
         callback();
     }
 
+    _applyFadeOutBehaviour(behaviour: Object, callback: () => mixed) {
+        const { colour, duration, id } = behaviour;
+        const overlayImageElement = document.createElement('div');
+        overlayImageElement.id = id;
+        this._setBehaviourElementAttribute(overlayImageElement, 'colour-overlay');
+        overlayImageElement.style.background = colour;
+        overlayImageElement.style.opacity = 0;
+        overlayImageElement.style.transition = `opacity ${duration}s`;
+        overlayImageElement.className = 'romper-image-overlay';
+        const startFade = () => { overlayImageElement.style.opacity = 1 };
+        this._target.appendChild(overlayImageElement);
+        this._behaviourElements.push(overlayImageElement);
+        setTimeout(startFade, 500);
+        callback();
+    }
+
+    _applyFadeInBehaviour(behaviour: Object, callback: () => mixed) {
+        const { colour, duration, id } = behaviour;
+        const overlayImageElement = document.createElement('div');
+        overlayImageElement.id = id;
+        this._setBehaviourElementAttribute(overlayImageElement, 'colour-overlay');
+        overlayImageElement.style.background = colour;
+        overlayImageElement.style.opacity = 1;
+        overlayImageElement.style.transition = `opacity ${duration}s`;
+        overlayImageElement.className = 'romper-image-overlay';
+        const startFade = () => { overlayImageElement.style.opacity = 0 };
+        this._target.appendChild(overlayImageElement);
+        this._behaviourElements.push(overlayImageElement);
+        setTimeout(startFade, 500);
+        callback();
+    }
+
+    // eslint-disable-next-line no-unused-vars
+    _applyFadeAudioOutBehaviour(behaviour: Object, callback: () => mixed) {
+        logger.warn('cannot fade out audio!');
```

Comment:
Why not? Change message to say why - i.e. 'X does not support fading audio'?

Reply:
So this will only be hit when you try to fade a non timed renderer, e.g., an image